server/hedley/modules/custom/hedley_news/plugins/content_types/news_content/news-content.tpl.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * @file
+ * News content template.
+ */
+?>
+
+<div class="row text-center">
+  <div class="col-xs-12">
+    <h2><?php print t('News and Updates'); ?></h2>
+  </div>
+</div>
+
+<div class="row text-center">
+  <div class="col-xs-12">
+    <h3><?php print $title; ?></h3>
+
+    <small>
+      <?php print t('Published on'); ?>
```

Comment:
this seems wrong -- it will not be translated correctly. Should be `t('Published on @data' , [...`